Search-Engine-Bandits/nutshell
# nav bar and css

# Description

Just the Nav Bar and CSS from Sophia. Logout NOT functional yet.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.




# Testing Instructions

git add and commit changes on your branch
git checkout master
git fetch --all
git checkout rc-navbar-sophia
observe



# Checklist:
